docs/src/guide/deployment/theming.md

```
@@ -119,10 +119,28 @@ docker run --rm -p 3001:3001 \
-The submit button should be red. Check the startup log if not: authup prints
-the resolved theme directory, whether the manifest loaded, and every file it
-will serve.
+The quickest check is `http://localhost:3001/logout`. It is served by the
+same console and carries the same theme, and it takes no parameters.
+
+To see the accent on the login form itself, open the hosted login page. Note
+that `/authorize` rejects an incomplete request by rendering an error card
+(with a `200`, so the page still loads), and every parameter below is
+required:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)